ui_testing/testcases/basic_tests/test04_testunits.py

```diff
@@ -1144,3 +1199,21 @@ def test031_editing_limit_of_quantification_fields_should_affect_table_and_versi
                 self.assertEqual(record_data['Quantification Limit Unit'], random_unit)
             version_counter = version_counter+1
             record_counter = record_counter+1
+
+    def test032_archive_quantifications_limit_field(self):
+        """
+        New: Test unit: Configuration: Limit of quantification Approach: Display the new fields in the configuration section 
+        ( Upper limit & lower limit & unit of  limit of quantification ) and I can archive them. 
+        User can archive the quantification limits field from the configuration section 
+
+        LIMS-4164
+        """
+        self.test_unit_page.open_configurations()
+        self.assertTrue(self.test_unit_page.archive_quantification_limit_field())
+        if self.base_selenium.check_element_is_exist(element='configurations_page:error_msg'):
+            self.base_selenium.LOGGER.info('this field is used in another testunit, you need to delete all testunits with quantification option to archive this field')
```

Comment:
All the time this test case will go inside this branch as we always have test units with quantifcation. I can see that Mazan developed the delete API, u can use it to delete all those test units then archive this field.

Reply:
@mazenmmbadr @dianaeldemerdash LIMS-4164 is on this branch only due to re-base from master, please if any of you can help with @0xIslamTaha  comment here

Reply:
@0xIslamTaha  this was my case, it is true currently, it will always go with the true branch, but if this case ran on a new data, where it has no testunits with quantification limit, the field will be deleted normally, and since we don't count on data, i made it with if condition to handle all situations